tests/test_astropyTableViews.py

```diff
@@ -127,9 +127,9 @@ def testFlagColumn(self):
         """Test that Flag columns can be viewed if copy=True or unviewable="copy".
         """
         v1 = self.catalog.asAstropy(unviewable="copy")
-        self.assertFloatsAlmostEqual(v1["a4"], self.catalog["a4"])
+        self.assertFalse(np.any(np.logical_xor(v1["a4"], self.catalog["a4"])))
```

Comment:
I wonder if `np.testing.assert_array_equal(v1["a4"], self.catalog["a4"])` (and similarly for the below) wouldn't be easier to interpret?